src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -40,7 +43,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="System\Diagnostics\Stopwatch.cs" />
-    <Compile Include="System\Environment.MachineName.cs" />
+    <Compile Include="System\Environment.MachineName.cs" Condition="'$(TargetGroup)' != 'uap'" />
```

Comment:
I think this one needs breaking out into two files, one for Unix that's excluded for non-unix and one for Windows that's okay for UAP and non UAP.  The test seems valid for UAP, at least for now (later we'll probably have to special case the test to expect funny results from Environment.MachineName on UAP but it certainly should be able to call it on UAP)